### DISCUSSION TITLE
General discussions

### DISCUSSION THREAD

**Discussion by codefaux** [codefaux/deemix-for-lidarr#1] [General]
Hey all. Assuming anyone winds up here, let me know what you think. Questions, comments, let me know. Raise an issue if there's a problem, or reach out here.

**Comment by jjeaton** [comment]
I mapped a config directory, but there are no files in it. How do I set the deemix config?

**Reply by jjeaton** [reply]
Actually, I fixed my issue by adding the Deemix delay profile. Now automatic search is working!

**Comment by OliverEricT** [comment]
@codefaux hello and thank you for extracting this. I noticed the setup has rss updates disabled, is there a specific reason for this? This would mean that only a manual press of the `Automatic/manual` search button would trigger a deemix download, correct?

**Reply by codefaux** [reply]
Hi there. You're welcome, I'm glad it's finding use.

To clarify regarding RSS; I haven't disabled RSS specifically, it just doesn't function when enabled so it's recommended to turn it off.

The background here is that Deemix does not provide an RSS feed for Lidarr to connect to. I presume this is because Deezer does not provide anything Deemix can use as an RSS feed. To my understanding, Deezer as as service doesn't really do tracking of content recently made available, so Deemix can't hook into that for Lidarr to poll against. Please note that it is also very possible that Deezer does provide such a data stream but it was not implemented in Deemix, but I'm not aware of a Deezer source for the information.

As a result, I believe you are correct; only an explicit search action will trigger a Deemix download. A workaround for this could be a simple script running in the background to do period searches via Lidarr API, based on various criteria. This is something I could be convinced to look into, if you were to articulate a clear set of rules by which to trigger searches.
